_pages/en_US/homebrew-launcher-(super-skaterhax).txt

```diff
@@ -0,0 +1,17 @@
+---
+title: "Homebrew Launcher (super-skaterhax)"
+---
+
+{% include_relative include/super-skaterhax.txt %}
```

Comment:
why is this an include when it is used nowhere else

Reply:
good question